client/src/pages/LessonHome/LessonHome.js

```diff
@@ -71,31 +72,36 @@ const LessonHome = ({ navigation }) => {
   useEffect(() => {
     const getData = async () => {
       if (lessonData?.vocab) {
-        // only attempt to display selected vocab items
         const selectedData = lessonData.vocab.filter((item) => item.selected)
-        const formattedVocabData = selectedData.map((item) => {
+        let formattedVocabData = selectedData.map(async (item) => {
+          const { fileURI: imageUri, shouldRefresh: shouldRefreshImage } = await getFileURI(item._id, 'image')
+          const { fileURI: audioUri, shouldRefresh: shouldRefreshAudio } = await getFileURI(item._id, 'audio')
+
           const formattedItem = {
             _id: item._id,
             name: item.original,
             body: item.translation,
-            audioURI: '',
+            audioURI: item.audio ? audioUri : '',
             audio: item.audio !== '',
             _order: item._order,
-            imageURI: '',
-            image: item.image,
+            imageURI: item.image ? imageUri : '',
+            image: item.image !== '',
           }
 
-          if (item.imageURI) {
-            formattedItem.imageURI = item.imageURI
-          } else if (item.image !== '') {
+          /*
+            Below, we only load the image and audio files from the API
+            if the file has an image and audio file AND it needs to be refetched from the API
+            because it 1) doesn't exist in Expo's file system or 2) has existed in Expo's file system for too long.
+          */
+
+          if (item.image !== '' && shouldRefreshImage) {
```

Comment:
Wait so` item.image` is a `string` but `formattedItem.image` is a boolean? Just want to make sure we are conciously making this change in the type.

Reply:
need to update naming for this